Android/ChoreCenterApp/app/src/test/java/com/cse403chorecenter/chorecenterapp/RewardHistoryTest.java

```diff
@@ -0,0 +1,46 @@
+package com.cse403chorecenter.chorecenterapp;
+
+import com.cse403chorecenter.chorecenterapp.ui.redeem_reward.RedeemRewardFragment;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.Assert.assertEquals;
+
+public class RewardHistoryTest {
```

Comment:
This test looks like it tests RedeemReward instead of RewardHistory as it uses RedeemRewardFragment.RewardModel instead of RewardHistoryFragment.RedeemedRewardModel. Earlier today I created a test also called RewardHistoryTest that has the same format as this but it uses the RedeemedRewardModel. I can push it for review.

Reply:
Ok sounds good.